airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -483,6 +483,18 @@ def streams(self, config: MutableMapping[str, Any]) -> List[Stream]:
                 sub_items_attr="refunds",
                 **args,
             ),
+            ParentIncrementalStipeSubStream(
+                name="customer_payment_methods",
```

Comment:
maybe a hot take, but I think we should just replace the existing `payment_methods` stream. It's not useful as far as I could tell